### ISSUE TITLE
CI fails for documentation build

### ISSUE THREAD

**Issue by katef** [katef/libfsm#411]
```
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/libx/libxml2/libxml2-utils_2.9.13%2bdfsg-1ubuntu0.2_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
```
this seems like not our fault, but we need some kind of fix

**Comment by katef** [comment]
Confirmed that this is fixed with just an `apt-get update`
https://github.com/katef/libfsm/actions/runs/4759800050/jobs/8461575200

**Comment by katef** [comment]
Fixed by #415
